recipes/librdkafka/all/conandata.yml

```diff
@@ -22,3 +22,6 @@ patches:
       base_path: source_subfolder
     - patch_file: patches/0003-Fix-broken-version-1-4-0-on-windows.patch
       base_path: source_subfolder
+  "1.5.2":
+    - patch_file: patches/0002-Change-library-targets-and-result-variables-1-5-2.patch
+      base_path: source_subfolder
```

Comment:
this patch can't properly work if you don't modify also upstream's top CMakeLists.